app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileDestination.kt

```diff
@@ -214,6 +218,73 @@ private fun ProfileScreen(
   }
 }
 
+@Composable
+private fun ColumnScope.ProfileRowsWithPlaceholders(
```

Comment:
I would dare say that we do not need to make this have in the name the `WithPlaceholders` part. That should just be a decision that it can make internally on how to render the profile rows according to the ProfileUiState passed into it.

And for the other `ProfileRows` further down which would then have the same signature, I would say that one can instead receive `ProfileUiState.Loaded` or something like that, indicating that there is no loading stuff anymore.

Since I know that the UiState does not have this kind of shape right now, perhaps it's the time to change it. If the UI wants those two states to be distinct enough as to what it will render, the presenter can help out by providing those states exactly like that instead.